lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -2083,6 +2085,84 @@ class CommandObjectSwift_RefCount : public CommandObjectRaw {
   }
 };
 
+class CommandObjectLanguageSwiftTaskBacktrace final
+    : public CommandObjectParsed {
+public:
+  CommandObjectLanguageSwiftTaskBacktrace(CommandInterpreter &interpreter)
+      : CommandObjectParsed(interpreter, "backtrace",
+                            "Show the backtrace of Swift tasks. See `thread "
+                            "backtrace` for customizing backtrace output.",
+                            "language swift task backtrace <variable-name>") {
+    AddSimpleArgumentList(eArgTypeVarName);
+  }
+
+private:
+  void DoExecute(Args &command, CommandReturnObject &result) override {
+    if (!m_exe_ctx.GetFramePtr()) {
+      result.AppendError("no active frame selected");
+      return;
+    }
+
+    if (command[0].ref().empty()) {
```

Comment:
Is command always guaranteed to be non-empty?

Reply:
thanks for catching this, it is not.